apollo-portal/src/main/resources/static/open/add-consumer.html

```diff
@@ -64,6 +64,27 @@ <h5>{{'Open.Manage.CreateThirdApp' | translate }}
                                     ng-bind="'Token: ' + consumerToken.token"></h4>
                             </div>
                         </div>
+
+                        <div class="form-group">
+                            <label class="col-sm-2 control-label">
+                                <apollorequiredfield></apollorequiredfield>
+                                {{'Open.Manage.Consumer.AllowCreateApplication' | translate }}
+                            </label>
+                            <div class="col-sm-3">
+                                <div>
+                                    <select
```

Comment:
How about displaying this as a checkbox?

![image](https://github.com/apolloconfig/apollo/assets/837658/6fafe77b-0965-4ad5-b637-984fae5cfa49)